src/components/EditMessageForm/EditMessageForm.js

```diff
@@ -123,24 +119,22 @@ class EditMessageForm extends React.Component {
 
   renderEmojiPicker = () => {
     const { t } = this.props;
-    if (this.props.emojiPickerIsOpen) {
-      return (
-        <div
-          className="str-chat__small-message-input-emojipicker"
-          ref={this.props.emojiPickerRef}
-        >
-          <Picker
-            native
-            emoji="point_up"
-            title={t('Pick your emoji')}
-            onSelect={this.props.onSelectEmoji}
-            color="#006CFF"
-            showPreview={false}
-            emojisToShowFilter={filterEmoji}
-          />
-        </div>
-      );
-    }
+    return this.props.emojiPickerIsOpen ? (
+      <div
+        className="str-chat__small-message-input-emojipicker"
+        ref={this.props.emojiPickerRef}
+      >
+        <Picker
+          native
+          emoji="point_up"
+          title={t('Pick your emoji')}
+          onSelect={this.props.onSelectEmoji}
+          color="#006CFF"
+          showPreview={false}
```

Comment:
What does this property do? I see we didn't have this set on the second emoji picker we had before, but we do here.

Reply:
The emoji-picker component we use has an option to show a slightly larger preview of the emoji with its name while you hover it. I don't think it really adds value, and we don't use it for the other MessageInput components. My guess is that it's just some leftover (like the double picker). I think this makes things a little bit more consistent across the board.